packages/vitaminjs-jest/src/babelrcJestTransformer.js

```diff
@@ -0,0 +1,13 @@
+import { createTransformer } from 'babel-jest';
+
+const babelConfig = {
+    presets: [
+        ['env', {
+            targets: { node: 'current' },
+        }],
+        'react'
```

Comment:
This `.babelrc` file above also has `stage-1` preset, but not `react` one, shouldn't they match?